CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## Version 1.1.0
+
+### Added
+Added additional validation to check if repository is versioned and show UI message that versioned repository is not supported".
```

Comment:
Remove " at the end. Remove the word UI.

Reply:
Done